app/experimenter/experiments/models/nimbus.py

```diff
@@ -80,3 +80,71 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class NimbusBucketNamespace(models.Model):
+    name = models.CharField(max_length=255)
+    instance = models.PositiveIntegerField(default=1)
+    total = models.PositiveIntegerField(default=ExperimentConstants.BUCKET_TOTAL)
+    randomization_unit = models.CharField(
+        max_length=255, default=ExperimentConstants.BUCKET_RANDOMIZATION_UNIT
+    )
+
+    class Meta:
+        verbose_name = "Bucket Namespace"
+        verbose_name_plural = "Bucket Namespaces"
+        unique_together = ("name", "instance")
+        ordering = ("name", "instance")
+
+    def __str__(self):  # pragma: no cover
+        return f"{self.name}-{self.instance}"
+
+    @classmethod
+    def request_namespace_buckets(cls, name, experiment, count):
```

Comment:
We discussed whether it's necessary to actually store a namespace for orthogonal experiments, but we figured it would be useful for the case of wanting to exclude another experiment that was added later.